docs/queries/byrole.mdx

```diff
@@ -321,6 +327,70 @@ To learn more about the `aria-level` property, see
 > The `level` option is _only_ applicable to the `heading` role. An error will
 > be thrown when used with any other role.
 
+### `value`
+
+A range widget can be queried by any value `getByRole('heading')` or by a
+specific value using the `level` option
+`getByRole('heading', { value: { now: 5, min: 0, max: 10, text: 'medium' } })`.
+
+Note that you don't have to specify all properties in `value`. A subset is
+sufficient e.g. `getByRole('heading', { value: { now: 5, text: 'medium' } })`.
+
+Given the example below,
+
+```html
+<body>
+  <section>
+    <button
+      role="spinbutton"
+      aria-valuenow="5"
+      aria-valuemin="0"
+      aria-valuemax="10"
+      aria-valuetext="medium"
+    >
+      Volume
+    </button>
+    <button
+      role="spinbutton"
+      aria-valuenow="3"
+      aria-valuemin="0"
+      aria-valuemax="10"
+      aria-valuetext="medium"
+    >
+      Pitch
+    </button>
+  </section>
+</body>
+```
+
+you can query the `Volume` spinbutton using
+`getByRole('heading', { value: { now: 5 } })`.
```

Comment:
I think would be better to reword this to avoid confusion with the other snippet that has more queries.

```suggestion
you can query specific spinbutton(s) with the following queries,
```